### PR TITLE
Don't show the admin layout to non-admins. Fixes #330

### LINKED ISSUE #330: Sign up page isn't using sufia layout
https://demo.hydrainabox.org/accounts/new

screen shot 2016-09-01 at 3 50 50 pm

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #330

### CODE REVIEW

File: app/controllers/accounts_controller.rb [hunk 1]
```diff
@@ -78,6 +78,14 @@ def destroy
 
   private
 
+    def decide_layout
+      if can? :read, :admin_dashboard
```
Comment (cbeer) [inline app/controllers/accounts_controller.rb]: Hm, would we ever want to show the `accounts#new` action using the admin template? Or maybe this is where the admin workflow and normal end-user workflow ought to diverge.

Comment (mjgiarlo) [inline app/controllers/accounts_controller.rb]: > Hm, would we ever want to show the `accounts#new` action using the admin template? 

Not that I can think up.

Comment (jcoyne) [inline app/controllers/accounts_controller.rb]: Yes. There's a "Create a new account" link on the Accounts index page (for admins)

Comment (jcoyne) [inline app/controllers/accounts_controller.rb]: See the blue button: 
screen shot 2016-09-01 at 5 06 44 pm

**Comment by cbeer** [comment]
Rebase?

**Comment by jcoyne** [comment]
Rebased.

**Pull request merged**